### DISCUSSION TITLE
Is there a way to change the order of columns

### DISCUSSION THREAD

**Discussion by jimbo108108** [RafaelGB/obsidian-db-folder#329] [Q&A] [answered]
I have a table with 4 columns and want to add a new column but would like to be able to make it the first column. Is there a way to do this?

**Comment by RafaelGB** [comment, marked as answer]
You have to eanble the DnD on global settings. There is an issue to fix a problem of compatibility with Obsidian https://github.com/RafaelGB/obsidian-db-folder/issues/239

**Reply by MJY1993** [reply]
Hi. I

1st. have the same issue. Drag and Drop not working. Where can i enable DnD on global settings? and what do you mean by 'global settings?' 
(I can drag and drop files to put into the other folders in obsidian)

2nd. after i delete columns and try to add a new filter, those deleted columns still appears as possible filter. Furthermore, those deleted columns(keys) and their values still exist in metadata in each of files. I thought when i delete columns on database, they would disappear not only on the database but also in metadata in each files.

**Reply by imed-ghomari** [reply]
[Documentation](https://rafaelgb.github.io/obsidian-db-folder/features/Columns/#reordering) for first issue.
[Documentation](https://rafaelgb.github.io/obsidian-db-folder/features/Columns/#removing) for second issue.

**Comment by jimbo108108** [comment]
Thanks for the quick response.
